tools/generate_rest.py

```diff
@@ -0,0 +1,33 @@
+import copy
```

Comment:
When and how is this script invoked?

Reply:
It should be invoked after taipy-rest and taipy-enterprise installed. How it is invoked should be similar to what we have done to taipy-core and taipy-gui ref manual, I am not really sure about this process.

Reply:
Ok so that would be part of the 'setup_generation' process.
How about adding a fourth step to the setup then?